codegen/templates/instances/Target0.sol.jinja

```diff
@@ -26,4 +26,10 @@ contract {{ name.capitalized }}Target0 is HyperdriveTarget0, {{ name.capitalized
     function kind() external pure override returns (string memory) {
         _revert(abi.encode({{ name.uppercase }}_HYPERDRIVE_KIND));
     }
+
+    /// @notice Returns the MultiToken's decimals.
+    /// @return The MultiToken's decimals.
+    function decimals() external pure override returns (uint8) {
+        _revert(abi.encode(uint8(18)));
+    }
```

Comment:
Do we want this in codegen? For many integrations, the default in HyperdriveTarget0 is fine. If anything, I'd say we use this when the yield source is payable since that indicates that the base token is ETH.

Reply:
fair. i'll take it out